File: backend/app/schemas/evidence.py
from typing import Optional, Any
from pydantic import BaseModel


class EvidenceItem(BaseModel):
    document_id: Optional[str] = None
    filename: Optional[str] = None
    page: Optional[int] = None
    source_text: str
    original_value: Optional[Any] = None
    new_value: Optional[Any] = None
    field: Optional[str] = None
    change_type: Optional[str] = None

File: frontend/src/types/evidence.ts
export interface EvidenceItem {
  document_id?: string | null;
  filename?: string | null;
  page?: number | null;
  source_text: string;
  original_value?: string | number | null;
  new_value?: string | number | null;
  field?: string | null;
  change_type?: string | null;
}
